src/Voron/Data/Tables/Table.cs

```diff
@@ -1680,6 +1680,25 @@ public long DeleteBackwardFrom(TableSchema.FixedSizeSchemaIndexDef index, long v
             return deleted;
         }
 
+        public bool FindByIndex(TableSchema.FixedSizeSchemaIndexDef index, long value, out TableValueReader reader)
```

Comment:
Given that this returns `bool` and actual result is `out` then I would rename the method to `TryFindByIndex` to follow the general convention